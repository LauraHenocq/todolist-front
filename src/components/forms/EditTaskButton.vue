<template>
  <v-container>
    <v-btn color="pink">Modifier la t√¢che</v-btn>
  </v-container>
</template>