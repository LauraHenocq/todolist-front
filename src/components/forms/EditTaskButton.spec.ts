import { mount, VueWrapper } from '@vue/test-utils';
import { beforeEach, expect, it, describe } from 'vitest'
import { createVuetify } from 'vuetify';
import EditTaskButton from './EditTaskButton.vue';

const vuetify = createVuetify();

describe('EditTaskButton', () => {
  let wrapper: VueWrapper

  beforeEach(() => {
    wrapper = mount(EditTaskButton, {
      global: {
        plugins: [vuetify],
      },
    });
  });

  describe('render', () => {
    it('should render', () => {
      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});