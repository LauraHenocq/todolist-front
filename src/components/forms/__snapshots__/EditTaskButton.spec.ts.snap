// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`EditTaskButton > render > should render 1`] = `
"<div class="v-container v-locale--is-ltr"><button type="button" class="v-btn v-btn--elevated v-theme--light bg-pink v-btn--density-default v-btn--size-default v-btn--variant-elevated"><span class="v-btn__overlay"></span><span class="v-btn__underlay"></span>
    <!----><span class="v-btn__content" data-no-activator="">Modifier la t√¢che</span>
    <!---->
    <!---->
  </button></div>"
`;
